css-view-transitions-1/Overview.bs

```diff
@@ -1662,6 +1662,17 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 			then |transition|'s [=ViewTransition/finished promise=] will reject with the same reason.
 	</div>
 
+## View transition page-visibility change steps ## {#page-visibility-change-steps}
+	<div algorithm>
+	The <dfn export>view transition page-visibility change steps</dfn> given {{Document}} |document| are:
+
+	1. If |document|'s [=Document/visibility state=] is "<code>hidden</code>",
```

Comment:
With my code hat on, should we also assert that if the visibility state change is "visible" then there is no active-view-transition. :)

Reply:
Added an assert.